docs/servo_power_monitor/src/index.ts

```diff
@@ -122,26 +147,33 @@ function setupStartUSBButton() {
     }
     if (!device) {
       device = null;
-      requestUSBButton.disabled = false;
       return;
     }
-
+    
```

Comment:
please delete this line (it seems to be irrerevant)

Reply:
clang-format can help you styling the code and fix issues like this automatically, please try it out ;)